processor/src/main/java/com/linecorp/decaton/processor/ProcessingContext.java

```diff
@@ -48,6 +49,12 @@
      */
     LoggingContext loggingContext();
 
+    /**
+     * @return The trace (returned by {@link TracingProvider}) associated with the task currently being
+     * processed. May be null.
+     */
+    Object trace();
```

Comment:
Can't be `TraceHandle` ?

Reply:
It's no longer actually needed - I'm thinking about removing it unless we have a use case for it? WDYT?

Reply:
+1. we can add it when it become necessary w/o breaking compatibility.

Reply:
I see. understood. then let's just remove for now